code/modules/clothing/masks/miscellaneous.dm

```diff
@@ -52,19 +52,6 @@
 	siemens_coefficient = 0.9
 	flags_armor_protection = BODY_FLAG_HEAD|BODY_FLAG_FACE|BODY_FLAG_EYES
 
-/obj/item/clothing/mask/horsehead
-	name = "horse head mask"
-	desc = "A mask made of soft vinyl and latex, representing the head of a horse."
```

Comment:
😐 Not something I expected to see here.

Reply:
It's in changelog and PR desc - that's a very old legacy item that isn't used in code and probably never was by admins, but bloats "speech problems" with snowflake handling. Keeping it would require refactoring them just for that.

Reply:
My comment wasn't trying to suggest keeping it.